applications/style.css

```diff
@@ -59,6 +59,13 @@ body {
   text-align: center;
 }
 
+.footer {
+  width: 100vw;
+  text-align: center;
+  position: fixed;
+  bottom: 4px;
+}
+
```

Comment:
Why you are repeating same class and property in different file

Reply:
Fixed it